server/database/sql/create_user_dashboard_basic.sql

```diff
@@ -0,0 +1,74 @@
+-- PROCEDURE: public.get_user_dashboard_basic(integer, character varying, refcursor, refcursor, refcursor)
+
+-- DROP PROCEDURE IF EXISTS public.get_user_dashboard_basic(integer, character varying, refcursor, refcursor, refcursor);
+
+CREATE OR REPLACE PROCEDURE public.get_user_dashboard_basic(
+	IN user_id integer,
+	IN running_avg character varying,
+	INOUT _words refcursor,
+	INOUT _counts refcursor,
+	INOUT _sentences refcursor)
```

Comment:
the three queries in this proc return a table of single words with all their data for the day they're used, a table of unique words with the number of times they were used, and a table of sentences with all the data from the day they  were used. this is pretty much what we use the google api for, all done in sql